### PR TITLE
#88: add license

### PR DESCRIPTION
**Pull request by bloerwald**
This repository didn't have a license so far, which generally is a bad idea.

This technically *changes* the license. Thus, technically, all prior contributors need to agree to the new license. The licenses chosen are permissible while also requiring attribution, with the purpose to not "steal" from anyone. 

Due to this being a potentially sensitive issue, I suggest there being a period of a week or two for past contributors to disagree. If you do, please reply to this pull request. In the best case, you also reply if you agree, so this doesn't take longer than it needs to. If there is no objection, I assume acceptance. I hope this doesn't hurt anyone.

- [x] @alinsavix 
- [ ] @barncastle 
- [x] @bloerwald 
- [x] @Deamon87 
- [x] @ferronn-dev 
- [x] @funjoker 
- [x] @Ghaster 
- [x] @gtker
- [ ] @HelloKitty 
- [x] @justMaku 
- [x] @liqourice 
- [x] @Marlamin 
- [x] @MaxtorCoder 
- [x] @mcd8604 
- [x] @mdX7 
- [x] @Meorawr 
- [x] @MMOSimca 
- [ ] @Razinao
- [x] @Shauren 
- [x] @skarnproject 
- [x] @tomrus88 
- [x] @vetyst

### CODE REVIEW

**Comment by Meorawr** [comment]
No objection from me.

**Comment by ferronn-dev** [comment]
Make it so.

**Comment by Ghaster** [comment]
Fine with me 👍

**Comment by funjoker** [comment]
I'm fine with it

**Comment by Shauren** [comment]
k

**Comment by liqourice** [comment]
Approved.

**Comment by MMOSimca** [comment]
Fine with me.

File: LICENSE.md
Comment (justMaku) [inline LICENSE.md:11]: nit: I'm not sure human beings can be licensed.

**Review by justMaku — APPROVED** [review]
a single nit, otherwise approved.

**Comment by vetyst** [comment]
I see no issue with it. Suprised you've to even ask permission after the smallest of contributions even.

**Comment by funjoker** [comment]
> I see no issue with it. Suprised you've to even ask permission after the smallest of contributions even.

That's how serious people handle it. They did that with DBCD the same way. The only reasonable way to do that ^^

**Comment by justMaku** [comment]
> I see no issue with it. Suprised you've to even ask permission after the smallest of contributions even.

For the license to be enforceable, it needs to be approved by every contributor. Adding a license without such approval could lead to legal issues.

**Comment by gtker** [comment]
Approved.

**Comment by mcd8604** [comment]
I approve of the new license.

**Comment by mdX7** [comment]
👍

**Comment by Deamon87** [comment]
Seems good! 👍

**Review by Deamon87 — APPROVED** [review]

File: LICENSE.md
Comment (bloerwald) [reply LICENSE.md:11]: I was more scared that Bavaria Media was going to complain. Maybe all of this is just a scam I'm doing to get fully permission to market the Smurf brand myself.

**Comment by bloerwald** [comment]
Friendly reminder for @alinsavix (1 commit), @barncastle (94 commits), @HelloKitty (1 commit) and @Razinao (2 commits) to refuse the license for another few days until Dec 26th before I assume approval.

**Comment by alinsavix** [comment]
Works for me.  👍

**Comment by bloerwald** [comment]
Two weeks passed, the licenses are very lax, and a strong majority voted to accept. 
/merge

**Pull request merged**
